crates/gql/src/types/filters.rs

```diff
@@ -99,7 +98,8 @@ impl GraphQLType<DefaultScalarValue> for NumericFilter {
         DefaultScalarValue: 'r,
     {
         let args = [
-            build_number_filter_argument(&spec.name, "isEqual"),
+            build_number_filter_argument(&spec.name, "equals"),
+            build_number_filter_argument(&spec.name, "notEquals"),
             build_number_filter_argument(&spec.name, "greaterThan"),
             build_number_filter_argument(&spec.name, "greaterOrEqual"),
             build_number_filter_argument(&spec.name, "lowerThan"),
```

Comment:
I think the GQL way to do this is `_eq`, `_neq`, etc, can we use those instead?

(I'm good with converting this comment to an issue and resolving later)

Reply:
hmm do you have a source? I've definitely seen different conventions.